tasks/master.yml

```diff
@@ -33,6 +33,24 @@
     owner: root
     mode: 0755
 
+- name: Stop BIND9 daemon
+  service:
+    name: bind
+    enabled: false
+    state: stopped
+
+# This oneliner must be split into separate port checks or refactored
+# in order to allow to control separate services like Kerberos, LDAP
+# and DNS.
+- name: Check if conflicting processes are running
```

Comment:
This looks like reinventing [fuser](http://linux.die.net/man/1/fuser) utility. What about reusing it instead, like this:

```yaml

name: terminate processes listening reserved ports
command:
  fuser -k -TERM -n tcp {{ port }}
loop:
  - 53
  - 389
  - 88
loop_control:
  loop_var: port
```